src/coreclr/src/vm/invokeutil.cpp

```diff
@@ -954,27 +949,28 @@ void InvokeUtil::SetValidField(CorElementType fldType,
     {
         pDeclMT = declaringType.GetMethodTable();
 
+        // We don't allow setting the field of nullable<T> (hasValue and value)
+        // Because you can't independantly set them for this type.
```

Comment:
A nit - independantly -> independently
```suggestion
        // Because you can't independently set them for this type.
```